tests/test_api_validate.py

```diff
@@ -3330,11 +3330,11 @@ def test_02_application_specific_password_token(self):
                                            method='POST',
                                            data={"user": "cornelius",
                                                  "service_id": "thunderbird",
-                                                 "pass": "test{0!s}".format(password)}):
```

Comment:
If the password has same evil chars this would probably also happen in line 3322.

Reply:
This probably slipped through since we just check if the authentication was `REJECT`ed, but not why.
Will fix this

Reply:
Done in 216198383